gpu_ocean/demos/sequential_importance_resampling.py

```diff
@@ -0,0 +1,357 @@
+# -*- coding: utf-8 -*-
```

Comment:
Python script that should be able to run SIR on a cluster of nodes using MPI. Look at mainLoop for what is actually going on . Most of the other functions are simply setup of initial conditions etc. and plotting.